textattack/transformations/word_swap_neighboring_character_swap.py

```diff
@@ -9,10 +9,16 @@ class WordSwapNeighboringCharacterSwap(WordSwap):
         Args:
             random_one (bool): Whether to return a single word with two characters 
                 swapped. If not, returns all possible options.
+            skip_first_char (bool): Whether to disregard perturbing the first 
+                character.
+            skip_last_char (bool): Whether to disregard perturbing the last 
+                character.
     """
-    def __init__(self, random_one=True, **kwargs):
+    def __init__(self, random_one=True, skip_first_char=False, skip_last_char=False, **kwargs):
```

Comment:
nit: Is there a reason we have separate skip_first_char and skip_last_char args rather than a replace_ends bool?